docs/platform/getting-started/overview.mdx

```diff
@@ -18,13 +18,8 @@ Your setup might be a little different depending on which solution you chose.
 Getting started with Falcon Platform can be broken down into a few simple steps.
 
 1. [Create a Falcon Platform account](prerequisites)
+2. [Setup Dcloud](dcloud)
 2. [Get access to our private NPM packages](npm)
```

Comment:
The numbering is incorrect because of this addition. It however doesn't appear to matter, because markdown (at least on github) automagically counts the list items itself. Might still be wise to fix it though.

Reply:
Done